docs/user/02-configuration.md

```diff
@@ -50,47 +50,92 @@ Use the following sample CRs as guidance. Each can be applied immediately when y
+| ---- | ----------- | ---- |
+| **activeBackend** (required) | string |  |
+| **conditions**  | \[\]object | Condition contains details for one aspect of the current state of this API Resource.
+---
+This struct is intended for direct use as an array at the field path .status.conditions.  For example,
+
+
+	type FooStatus struct{
+	    // Represents the observations of a foo's current state.
+	    // Known .status.conditions.type are: "Available", "Progressing", and "Degraded"
+	    // +patchMergeKey=type
+	    // +patchStrategy=merge
+	    // +listType=map
+	    // +listMapKey=type
+	    Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
+
+
+	    // other fields
+	} |
+| **conditions.&#x200b;lastTransitionTime** (required) | string | lastTransitionTime is the last time the condition transitioned from one status to another.
+This should be when the underlying condition changed.  If that is not known, then using the time when the API field changed is acceptable. |
+| **conditions.&#x200b;message** (required) | string | message is a human readable message indicating details about the transition.
+This may be an empty string. |
+| **conditions.&#x200b;observedGeneration**  | integer | observedGeneration represents the .metadata.generation that the condition was set based upon.
+For instance, if .metadata.generation is currently 12, but the .status.conditions[x].observedGeneration is 9, the condition is out of date
+with respect to the current state of the instance. |
+| **conditions.&#x200b;reason** (required) | string | reason contains a programmatic identifier indicating the reason for the condition's last transition.
+Producers of specific condition types may define expected values and meanings for this field,
+and whether the values are considered a guaranteed API.
+The value should be a CamelCase string.
+This field may not be empty. |
+| **conditions.&#x200b;status** (required) | string | status of the condition, one of True, False, Unknown. |
+| **conditions.&#x200b;type** (required) | string | type of condition in CamelCase or in foo.example.com/CamelCase.
+---
+Many .condition.type values are consistent across resources like Available, but because arbitrary conditions can be
+useful (see .node.status.conditions), the ability to deconflict is important.
+The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt) |
+| **publisherService**  | string |  |
+| **specHash** (required) | integer |  |
+| **state** (required) | string | Can have one of the following values: Ready, Error, Processing, Warning. Ready state is set
+when all the resources are deployed successfully and backend is connected.
+It gets Warning state in case backend is not specified, NATS module is not installed, or EventMesh secret is missing in the cluster.
+Error state is set when there is an error. Processing state is set if recources are being created or changed. |
```

Comment:
does this mean "backend not specified, NATs module not installed, Secret missing" are not errors? 
The explanation for "Warning" is very specific, and the explanation for "Error" is extremely vague. Can we improve that?